src/app.ts

```diff
@@ -148,7 +149,9 @@ app.get(
     withRequestMiddlewares(
       getApiClientMiddleware(),
       getUserFromRequestMiddleware(),
-      OptionalParamMiddleware("email", EmailString)
+      OptionalParamMiddleware("email", EmailString),
+      OptionalQueryParamMiddleware("offset", NonEmptyString),
+      OptionalQueryParamMiddleware("limit", NonEmptyString)
```

Comment:
Great! I will use `IntegerFromString.pipe(NonNegativeInteger)` because `NonNegativeIntegerFromString` isn't available in *ts-commons* package version used in the repo.